src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,60 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTests = codeWriter.getFlakyTests();
+
+      if (flakyTests.size() > 0) {
+        HashMap<TypedOperation, Integer> testOccurrences = new HashMap<>();
+        HashMap<TypedOperation, Integer> flakyOccurrences = new HashMap<>();
+
+        // Tally occurrences of all methods for if-idf
+        // Each method is counted once if it appears in a sequence test.
+        tallyOperationsInSequences(testOccurrences, explorer.getRegressionSequences());
+
+        // TODO: cxing handle Error Test Sequence tallying.
+        // tallyOperationsInSequences(testOccurrences, explorer.getErrorTestSequences());
+
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+        for (String flakyTestNum : flakyTests) {
+          int testNum = Integer.parseInt(flakyTestNum.substring(4)); // length of test
+          // Tests start at 001
+          ExecutableSequence flakyTest = regressionSequences.get(testNum - 1);
+          HashSet<TypedOperation> ops = new HashSet<>();
+          SimpleList<Statement> statements = flakyTest.sequence.statements;
+          for (int i = 0; i < statements.size(); i++) {
+            if (!statements.get(i).getOperation().isMethodCall()) continue;
+            ops.add(statements.get(i).getOperation());
+          }
+
+          for (TypedOperation to : ops) {
+            if (flakyOccurrences.containsKey(to)) {
+              flakyOccurrences.put(to, flakyOccurrences.get(to) + 1);
+            } else {
+              flakyOccurrences.put(to, 1);
+            }
+          }
+        }
+
+        Map<String, Double> methodFlakyPercentage = new HashMap<>();
```

Comment:
There is no documentation of what computation is being done.  A few words about that would make the code easier to understand, and then the specification and the implementation can be reviewed independently rather than reverse-engineering the code to determine what it does, then determining whether what it does is desirable.

Reply:
Putting values in this map, only to sort them later, feels like a waste.
You could use a priority queue instead, and then you wouldn't have to sort later.
(I would suggest a TreeMap from Double to String, but you want to permit two names to have the same ranking without losing them.)